### PR TITLE
Removed sending-metrics-to-gnocchi-and-to-stf m…

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
…odule

### CODE REVIEW

**Comment by JoanneOFlynn2018** [comment]
@leifmadsen Didn't delete the module itself in case we need any of the contents for Op Measures guide... WDYT?

**Comment by leifmadsen** [comment]
I have updated this branch from `master` to pull in the changes that fixes the syntax issue in the renewal procedure, and should allow the included STF removal procedure to be included again. Should also fix issues on downstream import.

File: doc-Service-Telemetry-Framework/master.adoc [hunk 1]
```diff
@@ -38,7 +38,7 @@ include::assemblies/assembly_advanced-features.adoc[leveloffset=+1]
 include::assemblies/assembly_renewing-the-amq-interconnect-certificate.adoc[leveloffset=+1]
 
 // upgrading to 1.4
-include::assemblies/assembly_upgrading-service-telemetry-framework-to-version-1-4.adoc[leveloffset=+1]
+//include::assemblies/assembly_upgrading-service-telemetry-framework-to-version-1-4.adoc[leveloffset=+1]
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/master.adoc:41]: 👍🏻

**Review by leifmadsen — APPROVED** [review]
I built this locally and reviewed. Everything looks good. Thanks!

**Pull request merged**